src/find_controls.cpp

```diff
@@ -236,7 +234,7 @@ FindMode::Values GetFindModeValueImpl()
 
 FindMode::Values GetFindModeValue()
 {
-    static const FindMode::Values val = GetFindModeValueImpl();
+    const FindMode::Values val = GetFindModeValueImpl();
```

Comment:
This would negatively affect performance and pollute log with messages from `GetFindModeValueImpl()`

Reply:
corrected log pollution, minor performance improvement.

Reply:
[resolved]